src/chains.ts

```diff
@@ -351,6 +351,231 @@ export const supportedChains: Array<Chain> = [
     },
   },
 
+  // 2 - Expanse Network
+  {
+    key: ChainKey.EXP,
+    name: 'Expanse Network',
+    coin: CoinKey.EXP,
+    id: 2,
+    visible: false,
+    tokenlistUrl: '', // TODO: find token list url
```

Comment:
Please leave `tokenlistUrl`undefined if we don't have one. It is an optional property.

Reply:
got it